backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/pporder/process/WEBUI_PP_Order_HUEditor_Launcher.java

```diff
@@ -173,7 +172,7 @@ private boolean isEligibleHuToIssue(final HuId huId)
 			return false;
 		}
 
-		if (huOrderCandidatesRepo.isHuIdIssued(huId))
```

Comment:
pls don't remove it

Reply:
Same reply as previous one: i'm checking `HU.Status` via `huStatusBL` instead

Reply:
Confirmed with teo on skype: It's ok to use HU.Status instead of using PP_Order_Qty.